### DISCUSSION TITLE
Reschedule φροντιστηρίου.

### DISCUSSION THREAD

**Discussion by elias2500** [courses-ionio/sw#934] [Announcements]
Λόγω αλλαγής στο πρόγραμμα η Πέμπτη 19:00-21:00 δεν είναι πλέον διαθέσιμη για όλους. Παρακάτω το νέο πρόγραμμα:
[Ωρολόγιο Πρόγραμμα ΣΤ' εξαμήνου - ακαδημαϊκό έτος 2020-2021 (1η ΟΡΘΗ ΕΠΑΝΑΛΗΨΗ).pdf](https://github.com/courses-ionio/sw/files/6184174/-.2020-2021.1.pdf)
Προτείνω την Δευτέρα 13:00-15:00. Παρακαλώ και  για τις απόψεις τον συναδέλφων. :D

**Comment by PaulosKal** [comment]
Οποιαδήποτε μέρα εκτός Παρασκευής είναι αποδεκτή από εμένα.

**Reply by cyclenikolaos** [reply]
Για μένα μια χαρά ακούγεται η Δευτέρα

**Reply by p13brif** [reply]
+1 για την Δευτέρα

**Reply by p18myro** [reply]
Νομίζω Δευτέρα είναι οκ

**Reply by ghost** [reply]
Καλύτερα Δευτέρα βοηθάει και εμάς που είμαστε μεγαλύτερο έτος

**Reply by p16pyla** [reply]
Τετάρτη κανείς ;

**Reply by epidrome** [reply]
ΟΚ, για Δευτ. 1-3μμ, πάντως από την μέχρι τώρα παρουσία σας καθώς και την ψηφοφορία εδώ το φροντιστήριο δεν φαίνεται να το θέλουν οι περισσότεροι

**Reply by epidrome** [reply]
@c15pati σε βολεύει το Δευτ 1-3μμ? 

εγώ μπορώ Τρι-Τετ 7-9μμ για όσους δεν έχουν εκείνα τα μαθήματα επιλογής

**Reply by c15pati** [reply]
Δευτέρα 1-3 θα προτιμούσα όχι. Τρίτη όμως μπορώ να βάλω έξτρα ώρες για να συμπληρώσω όσους δεν μπορούν την Δευτέρα.

**Reply by epidrome** [reply]
@c15pati έχεις δει το πρόγραμμα? μόνο Δευτ, Τετ, Παρ έχει ένα 2ώρο κενό

**Reply by apostoloskp** [reply]
Τελικά θα γίνει το φροντιστήριο από αύριο?

**Reply by c15pati** [reply]
@apostoloskp έχει βγει ανακοίνωση

**Reply by epidrome** [reply]
@c15pati για το φροντιστήριο που ρωτάει δεν βλέπω κάποια ανακοίνωση!? άλλο είναι το φροντιστήριο, άλλο είναι οι προσωπικές συναντήσεις!!

@apostoloskp από την παραπάνω συζήτηση προκύπτει ότι η διδακτική ομάδα δεν μπορεί Δευτ 1-3 αλλά επειδή πράγματι έχετε κενό σας προτρέπουμε (για μια ακόμη φορά!!!) να μπείτε και να κάνετε χρήση των εργαλείων

επίσης, όπως έγραψα πριν:

> εγώ μπορώ Τρι-Τετ 7-9μμ για όσους δεν έχουν εκείνα τα μαθήματα επιλογής

καταλαβαίνω ότι δεν τους βολεύει όλους, αλλά και όταν οι Πέμπτες ήταν διαθέσιμες και τους βόλευε όλους δεν είδαμε κάποια συμμετοχή

συνοπτικά, νομίζω πως το πρόβλημα δεν είναι οι διαθέσιμες ώρες αλλά η διάθεση συμμετοχής ;-) τα εργαλεία είναι πάντα διαθέσιμα!
